### PR TITLE
Enum matching popup should be wider

### PR DESCRIPTION
**Pull request by Abhinegi2**
closes: #2494 

### Visible/Frontend Changes

- [x] Enum popup will now be using fill wdth and wider.

### CODE REVIEW

**Review by sleidig — CHANGES_REQUESTED** [review]
We have a similar issue for entity-select dropdowns, e.g. see http://localhost:4200/attendance/recurring-activity -> details view -> assigned users

If there is no way to handle this in the underlying app-basic-autocomplete, we may need to implement something similar here as well ...

**Comment by Abhinegi2** [comment]
> We have a similar issue for entity-select dropdowns, e.g. see http://localhost:4200/attendance/recurring-activity -> details view -> assigned users
> 
> If there is no way to handle this in the underlying app-basic-autocomplete, we may need to implement something similar here as well ...

updated entity select component by using same approach

File: src/app/core/common-components/entity-select/entity-select.component.ts [hunk 1]
```diff
@@ -20,11 +20,13 @@ import { asArray } from "../../../utils/utils";
...
 import { FormDialogService } from "../../form-dialog/form-dialog.service";
 import { EntityRegistry } from "../../entity/database-entity.decorator";
-
 @Component({
   selector: "app-entity-select",
   templateUrl: "./entity-select.component.html",
-  styleUrls: ["./entity-select.component.scss"],
+  styleUrls: [
+    "./entity-select.component.scss",
+    "../../basic-datatypes/configurable-enum/enum-dropdown/enum-dropdown.component.scss",
```
Comment (sleidig) [inline src/app/core/common-components/entity-select/entity-select.component.ts:28]: you can move this stylesheet to the folder of the underlying basic-autocomplete

**Review by sleidig — CHANGES_REQUESTED** [review]
Your approach with positioning absolute has the disadvantage that the text (if long) overlaps with the icons, making both unreadable:
image

 Maybe we need to limit the width of the input then. Or work with content projection ...?
 Can you find out how `matIconSuffix` usually does that?

File: src/app/core/common-components/entity-select/entity-select.component.ts (hunk 1 above)
Comment (Abhinegi2) [reply src/app/core/common-components/entity-select/entity-select.component.ts]: it`s update please have a look now

**Review by sleidig — APPROVED** [review]
The icons are still overlapped in some edge cases but this seems good enough for now to close the issue :+1:

**Pull request merged**